README.md

```diff
@@ -61,14 +61,11 @@ Or if you prefer to run without docker (for speed + ability to integrate with de
 **Run with prod data**
 
 ```
-LOAD_DATA="prod" ./scripts/serve-local.sh
-LOAD_DATA="prod" ./scripts/undockered.sh
+./scripts/load_from_heroku joplin
 ```
 
--   This will add some seeding content from the last prod datadump (`joplin/db/system-generated/prod.datadump.json`) created by migration-test.sh.
--   LOAD_DATA and LOAD_PROD_DATA both load from the prod.datadump.json
--   You have to run the LOAD_x_DATA commands on a clean db instance. You can wipe the DB and load data in the same step by running:
-    -   `DROP_DB=on LOAD_DATA="prod" ./scripts/serve-local.sh`
```

Comment:
Doesn't the migration test script still work? I rarely use it, but its a workflow that still works if you want to load production data on a local instance that has new migrations

Reply:
oh i thought it wasnt working anymore. ill revert the README